activerecord/test/cases/date_time_precision_test.rb

```diff
@@ -191,6 +182,15 @@ def test_writing_a_blank_attribute
           assert_nil Foo.create!(happened_at: "").happened_at
         end
       end
+    else
+      def test_writing_a_blank_attribute
```

Comment:
Oops, this was my bad

Both tests should run in postgres, they should have different names.

Reply:
This is probably the fix we want https://github.com/rails/rails/pull/43992